src/pages/markets/instruments/_drift_switch_indices.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Symbol from '../components/helper/_symbol'
+import { drift_switch_indices } from '../static/content/_market-symbols'
+
+const DriftSwitchIndices = () => {
+    return (
+        <>
+            {drift_switch_indices.map((symbol, index) => (
+                <Symbol key={index} src={symbol.src} text={symbol.text} />
```

Comment:
Can remove the `index` and use` symbol.text` as key

Reply:
Sure